src/frontend/src/lib/components/send/SendSource.svelte

```diff
@@ -9,13 +9,15 @@
 
 	export let token: OptionToken;
 	export let balance: OptionBalance;
-	export let source: string;
+	export let source: string | undefined = undefined;
 </script>
 
-<Value ref="source" element="div">
-	<svelte:fragment slot="label">{$i18n.send.text.source}</svelte:fragment>
-	{source}
-</Value>
+{#if nonNullish(source)}
+	<Value ref="source" element="div">
```

Comment:
Given that I assume there will be some time before we can test this with E2E, should we provide a test for the change in the meantime, WDYT?

Reply:
@peterpeterparker If not an E2E test, what kind of test do you mean?

Reply:
I mean the vitest `npm run test`

Reply:
@peterpeterparker might be a good idea.

Reply:
@peterpeterparker I just added some test cases for the forms.